packages/core/strapi/package.json

```diff
@@ -66,9 +66,7 @@
   ],
   "main": "./lib",
   "types": "./lib/index.d.ts",
-  "bin": {
-    "strapi": "./bin/strapi.js"
-  },
+  "bin": "./bin/strapi.js",
```

Comment:
Are we sure about this one (scope package and all) ?

Reply:
This was an auto-change but I did miss this package is scoped. I'll look into reverting and investigating why `yarn` wants to change it (might be a one off)...

Reply:
it might just be working but I couldn't find a mention in npm docs about scope packages bins tbh

Reply:
Updated bin field is just fine!

See folders page:
https://docs.npmjs.com/cli/v9/configuring-npm/folders#executables

And scope:
https://docs.npmjs.com/cli/v9/using-npm/scope#installing-scoped-packages

Reply:
I've just done an experimental release `0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82c` and the `create-strapi-app` works as does things like `strapi-build` 👍🏼

Reply:
Mh, pnpm keeps telling me:

```
ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @strapi/plugin-users-permissions@0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82c
```

Same for all packages ... 🤔

Reply:
No idea, what did you run to get this? So i can repro.

Reply:
As soon as I declare a new project with

```
"dependencies": {
    "@strapi/strapi": "0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82c",
    "@strapi/plugin-users-permissions": "0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82c",
    "@strapi/plugin-i18n": "0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82c",
    "better-sqlite3": "8.0.1"
  },
```

and run either `npm i` or `pnpm i`. I can see the version on npm - maybe I am just blind this morning ... 🙈

Reply:
ah, i missed a letter 😂  `0.0.0-31bc1234844dc8acce70f8ae14d099dabb6a82cc`